lib/ClimaLandSimulations/src/utilities/data_tools.jl

```diff
@@ -0,0 +1,138 @@
+export replace_missing_with_mean!,
```

Comment:
We should make sure to incorporate the changes to this file from #430 before merging. Should be as simple as copying over this file from main and then changing the `VerifiedColumn` call to `filtercolumn`

Reply:
done!